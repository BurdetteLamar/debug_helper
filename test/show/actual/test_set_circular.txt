---
Set (message='test_set_circular'):
  Set#size: 1
  Element 0:
    Set:
      Set#size: 1
      Element 0: 'Set #<Set: {#<Set: {#<Set: {...}>}>}>'
