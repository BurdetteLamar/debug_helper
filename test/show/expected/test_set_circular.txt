---
Set (message='test_set_circular'):
  size: 1
  Element 0:
    Set:
      size: 1
      Element 0: 'Set #<Set: {#<Set: {#<Set: {...}>}>}>'
